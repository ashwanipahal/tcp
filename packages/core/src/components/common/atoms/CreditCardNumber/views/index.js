export { default } from './CreditCardNumber.view';
