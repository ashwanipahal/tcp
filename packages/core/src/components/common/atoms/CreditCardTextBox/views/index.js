export { default } from './CreditCardTextBox.native';
