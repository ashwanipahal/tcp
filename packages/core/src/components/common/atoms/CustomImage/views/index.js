export { default } from './CustomImage.view.native';
