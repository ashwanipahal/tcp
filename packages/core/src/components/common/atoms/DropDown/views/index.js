export { default } from './DropDown.native';
