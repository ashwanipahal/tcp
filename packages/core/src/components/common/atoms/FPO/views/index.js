export { default } from './FPO.view';
