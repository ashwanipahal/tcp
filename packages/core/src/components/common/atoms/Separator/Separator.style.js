import { css } from 'styled-components';

const styles = css`
  background-color: ${props =>
    props.backgroundColor ? props.backgroundColor : props.theme.colors.BLACK};
  height: ${props => (props.height ? props.height : '3px')};
  border: none;
  margin-bottom: 40px;
`;

export default styles;
