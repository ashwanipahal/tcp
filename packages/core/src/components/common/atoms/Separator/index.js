// @flow
export { default } from './views/Separator';
