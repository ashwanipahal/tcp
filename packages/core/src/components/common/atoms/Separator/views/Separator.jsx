// @flow
import React from 'react';
import withStyles from '../../../hoc/withStyles';
import styles from '../Separator.style';

const Separator = ({ className }: { className: string }) => <hr className={className} />;

export default withStyles(Separator, styles);
export { Separator as SeparatorVanilla };
