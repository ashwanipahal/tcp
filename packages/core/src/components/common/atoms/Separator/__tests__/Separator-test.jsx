import React from 'react';
import { shallow } from 'enzyme';
import { SeparatorVanilla } from '../views/Separator';

describe('Separator component', () => {
  it('renders correctly', () => {
    const props = {
      className: 'sample-class',
    };
    const component = shallow(<SeparatorVanilla {...props} />);
    expect(component).toMatchSnapshot();
  });
});
