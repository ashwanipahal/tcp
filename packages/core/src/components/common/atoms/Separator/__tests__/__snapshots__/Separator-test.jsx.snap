// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Separator component renders correctly 1`] = `
<hr
  className="sample-class"
/>
`;
